### ISSUE TITLE
Docker image for Concourse tasks with om-linux 0.37?

### ISSUE THREAD

**Issue by aegershman** [ECSTeam/docker-images#10]
**tl;dr** 
I'm a wandering open-source freeloader looking for a Docker patron, will you support a Docker image for use with Concourse tasks that [has `om-linux 0.37`](https://github.com/pivotal-cf/om/releases/tag/0.37.0)?

**background**
[I have been working on an `upgrade-tiles` project which uses selective deploys for new product versions and stemcells](https://github.com/aegershman/upgrade-tiles). This project requires [a feature](https://github.com/pivotal-cf/om/issues/158) in the 0.37 release of `om-linux`. Unfortunately, even though the release has been cut, there are no well-maintained docker images which I could use for Concourse tasks which contain the latest `om` version. Although I know I _can_ build my own Docker image with the latest `om` version, I would prefer if someone reputable in the OSS community who already maintains images & also has a vested technical interest / expertise in this would be willing to do it. I have asked some other groups if they would do this, but unfortunately, and understandably, they declined. No big deal:

- https://github.com/pivotal-cf/pcf-pipelines/issues/331
- https://github.com/pivotal-cf/om/issues/161

(And obviously, if you decide not to maintain this, absolutely no big deal at all.)
Thank you for your consideration!

**Comment by aegershman** [comment]
I'm going to go with providing the released om binary as an input to my pipeline tasks. I think that'll be better in the long run 👍
